tests/aws/services/sns/test_sns.py

```diff
@@ -1105,7 +1111,7 @@ def test_redrive_policy_lambda_subscription(
         snapshot.match("messages", response)
 
     @markers.aws.validated
-    @pytest.mark.parametrize("signature_version", ["1", "2"])
+    @pytest.mark.parametrize("signature_version", ["2"])
```

Comment:
is there a specific reason why this was changed?

Reply:
My bad, this was an accidental commit. I have updated it now, sorry for the confusion!